app/views/content_items/fatality_notice.html.erb

```diff
@@ -0,0 +1,46 @@
+<%= content_for :page_class, @content_item.format.dasherize %>
+<%= content_for :title, "#{@content_item.page_title} - #{t("content_item.format.#{@content_item.document_type}", count: 1)}" %>
+<%= content_for :meta_description, @content_item.description %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/title',
+        context: "Operations in #{@content_item.field_of_operation.title}",
+        title: @content_item.title %>
+  </div>
+</div>
+
+<%= render 'shared/withdrawal_notice', content_item: @content_item %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/metadata',
+        from: @content_item.from,
+        first_published: @content_item.published,
+        last_updated: @content_item.updated,
+        see_updates_link: true,
+        other: {
+          "Field of operation" => link_to(@content_item.field_of_operation.title, @content_item.field_of_operation.path)
+        }
+      %>
+  </div>
+</div>
+
+<%= render 'shared/description', description: @content_item.description %>
+
+<div class="grid-row sidebar-with-body">
+  <div class="column-third">
+    <%= render 'shared/sidebar_image',
+        image: { "url" => asset_path("ministry-of-defence-crest"), "alt_text" => "" } %>
+  </div>
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/govspeak', content: @content_item.body %>
+  </div>
+</div>
+
+<%= render 'govuk_component/document_footer',
+    from: @content_item.from,
+    published: @content_item.published,
```

Comment:
This is the existing Whitehall behaviour.

<img width="483" alt="screen shot 2016-09-20 at 16 50 44" src="https://cloud.githubusercontent.com/assets/2715/18677944/6ac5d07c-7f52-11e6-989f-778a015b71ac.png">

<img width="695" alt="screen shot 2016-09-20 at 16 50 37" src="https://cloud.githubusercontent.com/assets/2715/18677945/6ace2290-7f52-11e6-967b-70b9ce9b6a8b.png">

See https://www.gov.uk/government/fatalities/flight-lieutenants-alan-scott-and-geraint-roberts-have-died